packages/fast-components-styles-msft/src/text-field/index.ts

```diff
@@ -0,0 +1,74 @@
+import { IDesignSystem } from "../design-system";
+import { ComponentStyles, ICSSRules } from "@microsoft/fast-jss-manager";
+import { ITextFieldClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import { toPx } from "@microsoft/fast-jss-utilities";
+import { applyType } from "../utilities/typography";
+import * as Chroma from "chroma-js";
+
+function applyMixedColor(incomingProperty: string, mixValue?: number, alpha?: number): ICSSRules<IDesignSystem> {
```

Comment:
`incomingProperty` isn't really doing anything for this function but reducing extensibility. Instead, you could change to:

```
function applyMixedColor(minvalue?: number, alpha?: number): (config: IDesignSystem) => string {
    return function applyColors(config: IDesignSystem): string {
       if (alpha) {
           return Chroma.mix(config.foregroundColor, config.backgroundColor, mixValue).alpha(alpha).css();
       } else {
           return Chroma.mix(config.foregroundColor, config.backgroundColor, mixValue).css();
       }
   }
}
```

and later apply it:
```
{
    outlineColor: applyMixedColor(0.38)
}
```

Doing that would expand the scope of use of the function and reduce code complexity